src/main/java/org/everit/osgi/dev/maven/util/DistUtil.java

```diff
@@ -33,7 +33,11 @@ private String escape(final String value) {
     if (value == null) {
       return null;
     }
-    return value.replace(",", "\\,").replace("=", "\\=").replace(" ", "\\ ");
+    return value
+        .replace(",", "\\,")
```

Comment:
Backslash is escaped at line 80 and 84. These are the remaining non-escaped characters by the PropertiesEscape.